.github/workflows/on_pull_request_comments.yml

```diff
@@ -0,0 +1,24 @@
+name: Pull Request Comments
+
+on:
+  issue_comment:
+    types: [ created, edited ]
+
+jobs:
+  run-extended-tests:
+    if: github.event.issue.pull_request && contains(github.event.comment.body, '/run-extended-tests')
```

Comment:
Curious how does this `issue` works? Does it mean that every comment will trigger the workflow re-run?

Reply:
I think the work flow will be checked but nothing will be run.